source/tutorial/enforce-keyfile-access-control-in-existing-sharded-cluster-no-downtime.txt

```diff
@@ -0,0 +1,297 @@
+created in these steps are cluster-level users and cannot be used for
+accessing individual shard replica sets.
+
+.. include:: /includes/steps/enable-authentication-in-shardcluster-nodowntime-uac.rst
+
+Restart Each ``mongos`` Instance with ``transitionToAuth`` set to ``true``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Repeat these steps for each :program:`mongos` in the sharded cluster,
+working on one :program:`mongos` at a time.
+
+.. include:: /includes/steps/enable-authentication-in-shardcluster-nodowntime-transition-mongos.rst
+
+At the end of this section, all :program:`mongos` instances in the sharded
+cluster are running with :setting:`security.transitionToAuth` and
+:setting:`security.keyFile` internal authentication.
+
+Restart Each Config Server Replica Set Member with ``transitionToAuth`` set to ``true``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Repeat these steps for each :program:`mongod` in the config server replica
+set, one member at a time. Work on the :term:`secondary` or :term:`arbiter`
+members *first*, and the :term:`primary` *last*.
+
+.. include:: /includes/steps/enable-authentication-in-shardcluster-nodowntime-transition-config.rst
+
+At the end of this section, all :program:`mongod` instances in the config
+server replica set is running with :setting:`security.transitionToAuth` and
+:setting:`security.keyFile` internal authentication.
+
+Restart Each Member of Each Shard Replica Set with ``transitionToAuth`` set to ``true``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Create the shard-local administrator
+++++++++++++++++++++++++++++++++++++
+
+Each shard replica set in a sharded cluster that enforces authentication
+should have its own shard-local administrator. You cannot use a shard-local
+administrator for one shard to access another shard or the sharded cluster.
+
+At the completion of this tutorial, if you want to connect to the shard for
+performing maintenance operations on the shard replica set, you must
+authenticate as the shard-local administrator.
+
+Connect to the :term:`primary` member of the shard replica set and create a
+user with the :method:`db.createUser()` method, assigning it the following
+roles:
+ 
+- :authrole:`clusterAdmin` on the ``admin`` database
+- :authrole:`userAdmin` roles on the ``admin`` database
+
+.. code-block:: javascript
+
+    admin = db.getSiblingDB("admin")
+    admin.createUser(
+      {
+        user: "admin",
+        pwd: "<password>",
+        roles: [ 
+          { role: "clusterAdmin", db: "admin" },
+          { role: "userAdmin", db: "admin" } 
+        ]
+      }
+    )
+
+.. note::
+
+  Only connect to individual shard replica sets for performing
+  maintenance operations. Connect to a :program:`mongos` for
+  performing queries or cluster-level maintenance operations.
+
+Procedure
++++++++++
```

Comment:
If I see Procedure, I could skip over all the words above to just get to the procedure that has steps. 
If we change this section title, then this subheading could be the 
Restart shards with auth (or whatever)